.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,20 @@
-## Why are you doing this?
+<!-- all sections optional, delete any you don't need -->
+## What are you doing in this PR?
 
 <!--
-Remember, PRs are documentation for future contributors.
-
-If this PR is a fix, please include a link to the original PR that introduced
-the breakage for reference.
+This pr adds a widget to the doogle so that we can buy a sub from any page in one click
+For detailed changes see the inline self-review comments.
 -->
 
-[**Trello Card**](https://trello.com)
-
-## Changes
+## Why are you doing this?
```

Comment:
I did a quick audit of recent PRs (other than mine) and only one of them actually wrote the WHY, and usually the WHAT was written under the WHY title.  The WHY was usually not included at all.
So it makes sense to make the top title WHAT, and put the WHY below as an optional section.